Makefile

```diff
@@ -48,35 +46,22 @@ install-docs: docs
 
 install: install-docs
 
-cross: vendor
-	GOOS=windows $(MAKE) build-internal BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_WINDOWS_CROSS)"
-	GOOS=darwin $(MAKE) build-internal BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_DARWIN_CROSS)"
+cross:
+	GOOS=windows $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_WINDOWS_CROSS)"
+	GOOS=darwin $(MAKE) build BUILDTAGS="$(BUILDTAGS) $(BUILD_TAGS_DARWIN_CROSS)"
 
 tools: tools.timestamp
 
 tools.timestamp: Makefile
-	@go get -u $(BUILDFLAGS) golang.org/x/lint/golint
-	@go get $(BUILDFLAGS) github.com/vbatts/git-validation
-	@go get -u github.com/rancher/trash
+	@GO111MODULE="off" go get -u $(BUILDFLAGS) golang.org/x/lint/golint
+	@GO111MODULE="off" go get $(BUILDFLAGS) github.com/vbatts/git-validation
 	@touch tools.timestamp
 
-vendor: tools.timestamp vendor.conf
```

Comment:
I very much prefer keeping the vendor target. All other projects on `github.com/containers` do something as follows:

https://github.com/containers/skopeo/blob/481bb94c5f8e65bc83b97539e7e4e6ccb8428a73/Makefile#L178-L182

Reply:
I also want to preserve the having the code in `./vendor` (via `go mod vendor`). The workflow of many folks is used to that. If some don't want it, they are free to not use `make vendor`.